samples/bluetooth/mesh/light_ctrl/README.rst

```diff
@@ -249,6 +249,37 @@ Configure the Sensor Setup Server model on the **Mesh Sensor** node:
 
 The Sensor Setup Server model is now configured and able to receive sensor setting messages from the Sensor Client.
 
+Occupancy Mode
+--------------
+
+You can combine this sample with the :ref:`bluetooth_mesh_sensor_server` sample to trigger the :ref:`Occupancy On <bt_mesh_light_ctrl_srv_on_event>` event on the Light LC Server by the Occupancy Sensor.
```

Comment:
```suggestion
You can combine this sample with the :ref:`bluetooth_mesh_sensor_server` sample to trigger the Occupancy :ref:`On event <bt_mesh_light_ctrl_srv_on_event>` on the Light LC Server by the Occupancy Sensor.
```

Reply:
I think should be "Occupancy On" event according to the spec.

Reply:
Hmm I agree, perhaps the linked headline should be changed? (in `doc/nrf/libraries/bluetooth_services/mesh/light_ctrl_srv.rst`)

Reply:
I took the Occupancy On event out of On event in the light_ctrl_srv.rst. Please have a look.

Reply:
Looks good, much more clear now :)